G/glmnet/build_tarballs.jl

```diff
@@ -3,26 +3,33 @@
 using BinaryBuilder, Pkg
 
 name = "glmnet"
-version = v"5.0.0"
+version = v"4.0.2"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/JuliaStats/GLMNet.jl.git", "d8cab55556a8c184a879446476efcc35e68a3eee"),
+    GitSource("https://github.com/cran/glmnet.git", "b1a4b50de01e0cd24343959d7cf86452bac17b26")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/GLMNet.jl/deps/
+cd $WORKSPACE/srcdir/glmnet/src
+
+# Add stub for `setpb`, which normally comes from `pb.c` to connect the 
+# progress meter to R, but we don't need that
+echo "
+      subroutine setpb(val)
+      return
+      end
+" > pb.f
 
 flags="-fdefault-real-8 -ffixed-form -shared -O3"
 if [[ ${target} != *mingw* ]]; then
-    flags="${flags} -fPIC"
+    flags="${flags} -fPIC";
 fi
 if [[ ${target} != aarch64* ]] && [[ ${target} != arm* ]]; then
-    flags="${flags} -m${nbits}"
+    flags="${flags} -m${nbits}";
 fi
-
-${FC} ${LDFLAGS} ${flags} glmnet5.f90 -o libglmnet.${dlext}
+${FC} ${LDFLAGS} ${flags} glmnet5dpclean.f wls.f pb.f -o libglmnet.${dlext}
 
 mkdir -p $libdir
 mv libglmnet.${dlext} $libdir/
```

Comment:
This can be simplified to
```suggestion
mkdir -p ${libdir}
${FC} ${LDFLAGS} ${flags} glmnet5dpclean.f wls.f pb.f -o ${libdir}/libglmnet.${dlext}
install_license DESCRIPTION
```
I've also added `DESCRIPTION` as license file.  Not ideal, but it isn't worse than what upstream does

Reply:
Sounds good, thanks